share/mk/bsd.sys.mk

```diff
@@ -199,9 +199,12 @@ CWARNFLAGS+=	-Wno-error=pass-failed
 .endif
 
 .if ${COMPILER_TYPE} == "clang" && ${COMPILER_VERSION} >= 100000
-CWARNFLAGS+=	-Wno-error=misleading-indentation
-# CXXWARNFLAGS+=	-Wno-error=deprecated-copy
-CXXWARNFLAGS+=	-Wno-deprecated-copy
+# CXXWARNFLAGS+=	-Wno-deprecated-copy
+CXXWARNFLAGS+=	-Wno-error=deprecated-copy
+.endif
+
+.if ${COMPILER_TYPE} == "clang" && ${COMPILER_VERSION} >= 115000
```

Comment:
As I have said on Slack, upstream will presumably either fix these warnings or use `>= 110000`. We should do one of those, rather than our own weird incompatible thing. If we _really_ want to keep supporting the older 11 snapshot then we could make the warning's presence a compiler feature, but I strongly believe we don't want to be inventing our own version numbers that lie about being newer than they really are.